01_01_Basic_Rules.txt

```diff
@@ -628,6 +626,33 @@ Class Feats:
 specifies a class feat. Characters can not take non-class feats instead of class feats 
 or vice-versa. Class Feats are listed in the Classes document.
 
+==============================
+Items and Buying Items
+==============================
+
+	When you build a character, you will get some starting money and the time to
+spend it to acrue the kit that you'll start playing with. But this likely won't
+be the last time that you'll have the opportunity to buy things. It may be smart
+to save some of your starting cash to spend elsewhere. Some stores and vendors
+you meet will sell exactly what you see in the Player Handbook. Some traders 
+may offer items or weapons that aren't available either at the start of the game
+or anywhere else in the galaxy! 
+
+	When you buy a gun, the gun usually comes with 5 magazines of ammunition
```

Comment:
5 is many, since we've usually been playing with 2 or 3. Any particular reason?

Reply:
That's what 1Source had as his default at one point? I'd like players not to be so ammo-conscious that they abstain from suppression. But my heart isn't set on that number by any means.

Reply:
Is that a problem we can solve instead by separating costs for magazines of ammo and bulk ammo (with competitiveTM bulk pricing)?